pallets/restricted-tokens/src/lib.rs

```diff
@@ -509,20 +530,24 @@ pub mod pallet {
 				let old_reserved =
 					<T::Fungibles as fungibles::InspectHold<T::AccountId>>::balance_on_hold(
 						currency_id,
+						&(),
 						&who,
 					);
 				<T::Fungibles as fungibles::MutateHold<T::AccountId>>::release(
 					currency_id,
+					&(),
 					&who,
 					old_reserved,
-					false,
+					Precision::BestEffort,
```

Comment:
@NunoAlexandre Any particular reason why we go with `BestEffort` here whereas for the rest we have `Precision::Exact`? Asking because benchmarks are failing with `Account cannot be created for recording amount on hold`.

Reply:
@wischli it may have been a mistake in this case. Before we passed a `bool` where `true` meant `best_effort`, so if this was `false` before this should probably be `Exact`. Do the tests succeed with that option?